### PR TITLE
Address privacy property, #300

### PR DESCRIPTION
**Pull request by tfpauly**
Address #300

### CODE REVIEW

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -882,6 +882,29 @@ requiring specific PvD instances or interface instances, and should be preferred
...
+: Enumeration
+
+This property specifies whether Listeners and Connections should prefer the use of temporary addresses when possible.
+This is generally used to prevent linking connections over time when a stable address is not needed. Possible values are:
+
+Stable:
+: Prefer the use of stable (sometimes called "permanent") local addresses
+
+Temporary:
+: Prefere the use of temporary (sometimes called "privacy") addresses {{!RFC4941}}
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md]: typo s/Prefere/Prefer

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -882,6 +882,29 @@ requiring specific PvD instances or interface instances, and should be preferred
...
+
+Default:
+: Prefer the use of stable local addresses for Listeners and Rendezvous Connections, and prefer the use of temporary addresses for other Connections.
+
+The default is Default, which adapts based on use case.
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md]: I would remove this sentence - obviously "Default" is the default. We should generally avoid this repetition whenever we have "Default" as a value choice IMO.

File: draft-ietf-taps-interface.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-taps-interface.md]: Fixed!

File: draft-ietf-taps-interface.md (hunk 2 above)
Comment (tfpauly) [reply draft-ietf-taps-interface.md]: Removed!

**Pull request merged**